### DISCUSSION TITLE
How to include Mermaid Diagrams in HTML pages?

### DISCUSSION THREAD

**Discussion by davidljung** [requarks/wiki#3836] [Help / Questions]
I have the Mermaid (& Kroki) plugins enabled and can use a Kroki Mermaid diagram in a Markdown page, but how to I place a Mermaid diagram in a HTML page (one created with the HTML editor)?

(All our pages are Markdown except for the Home Page, since I selected HTML for maximum flexibility during installation and it isn't possible to switch it to Markdown now)

Thanks.

**Comment by Phil-hostwiki** [comment]
You can switch editors on existing pages, by using the Convert page action in the latest Wiki.js version (2.5.201).
https://docs.requarks.io/en/editors#change-editor

**Reply by davidljung** [reply]
That's the version I'm using, but the Page Actions popup doesn't seem to have any option on any of the tab that I can see.  Where is the Convert page function exactly?

**Reply by Phil-hostwiki** [reply]
![IMG_20210414_232704](https://user-images.githubusercontent.com/82508256/114788409-4ccc6780-9d79-11eb-9109-9a79b9ef6165.jpg)

**Reply by davidljung** [reply]
Thanks - the issue was I was mistaken about my version - it is .197 not .201.  I still can't figure out how to include Mermaid into a HTML page though.
